### PR TITLE
feature/GSYE-446: Add calculation of asset volume time series

### PR DESCRIPTION
**Pull request by mrsaemir**
As requested by GSYE-446, we want to add calculation of asset volume time series for forward markets.

## Proposed changes:
- Add `ForwardResultsHandler._generate_asset_volume_time_series`
- Add support for INTRADAY market to `ForwardTradeProfileGenerator`
- [x] Add unit tests

### CODE REVIEW

File: gsy_framework/sim_results/electric_blue/all_results.py
Comment (hannesdiedrich) [inline gsy_framework/sim_results/electric_blue/all_results.py:47]: NIT: Could be refactored into a separate method

File: gsy_framework/sim_results/electric_blue/volume_timeseries.py
Comment (hannesdiedrich) [inline gsy_framework/sim_results/electric_blue/volume_timeseries.py:96]: Did you make `asset_time_series_buffer` public in order to remove this method here and implement savinf to the DB on the gsy-web side?

**Review by hannesdiedrich — COMMENTED** [review]
Will wait for the gsy-web side to leave my approval. LGTM already though. Thanks!

File: gsy_framework/sim_results/electric_blue/volume_timeseries.py
Comment (mrsaemir) [reply gsy_framework/sim_results/electric_blue/volume_timeseries.py:96]: You're right. I made it public to access it from gsy-web.

File: gsy_framework/sim_results/electric_blue/all_results.py
Comment (mrsaemir) [reply gsy_framework/sim_results/electric_blue/all_results.py]: Nice suggestion. Fixed it.

**Review by hannesdiedrich — APPROVED** [review]
LGTM

File: gsy_framework/sim_results/electric_blue/all_results.py
Comment (spyrostz) [inline gsy_framework/sim_results/electric_blue/all_results.py:49]: Minor question, how come this is not an int ? IIRC this method is called in the simulation_results service, therefore it receives serialized dicts and not objects, therefore it will probably not contain the enum but the enum value (therefore an int)

Comment (mrsaemir) [reply gsy_framework/sim_results/electric_blue/all_results.py:49]: You're correct, however the problem is that integers can not be used as keys in JSON, so the serializer automatically casts it to a string.

```
In [1]: my_dict = {3: 'GSY'}                                                    

In [2]: import json                                                             

In [3]: json.dumps(my_dict)                                                     
Out[3]: '{"3": "GSY"}'
```

Comment (spyrostz) [reply gsy_framework/sim_results/electric_blue/all_results.py:49]: Got it. Thanks for the clarification!

**Review by spyrostz — APPROVED** [review]
LGTM

**Pull request merged**
